playground/src/pages/CodingEnvironment.tsx

```diff
@@ -221,15 +236,46 @@ export default function CodingEnvironment(): JSX.Element {
     });
   };
 
+  /**
+   * Shows the proof that is stored in the session.
+  */
+  const showProof = async () => {
+    if (!proof) {
+      setOutput("There is no proof to show. \nDid you prove the program?");
+      toast.error("Showing proof failed");
+      return;
+    }
+    toast.custom((t) => (
+      <div
+        className={`${
+          t.visible ? 'animate-enter' : 'animate-leave'
+        } max-w-full w-full bg-white shadow-lg rounded-lg flex ring-1 ring-black ring-opacity-5 overflow-hidden`}
+      >
+        <div className="flex-1 w-0 p-4 break-normal overflow-hidden">
+          <h1 className="text-lg font-medium text-gray-900"> This is your proof as Uint8Array </h1>
```

Comment:
`Uint8Array` is a JS specific thing that is unrelated to Miden, it is not useful for the user to know this.

```suggestion
          <h1 className="text-lg font-medium text-gray-900"> This is your proof </h1>
```

Reply:
done